public/app/common/services/devices/devices-service/devices-service.js

```diff
@@ -27,15 +27,22 @@ function extractThings (sirenResponse) {
 
 function initDeviceStreamById (deviceId) {
   return Rx.Observable
-    .fromPromise(fetch(thingUrl + '/' + deviceId + '?istr=true').then(extractJson));
+    .fromPromise(fetch(thingUrl + '/' + deviceId + '?thingAs=representation').then(extractJson));
 }
 
+var anchor = document.createElement('a');
+
+var stripHostname = function stripHostname (url) {
+  anchor.href = url;
+  return anchor.pathname;
+};
+
```

Comment:
It can and should, but we don't have any currently. Do you think we should start a new module for that?